packages/loader/container-loader/src/index.ts

```diff
@@ -20,3 +20,4 @@ export {
 	resolveWithLocationRedirectionHandling,
 } from "./location-redirection-utilities";
 export { IProtocolHandler, ProtocolHandlerBuilder } from "./protocol";
+export { parseIResolvedUrlUrlIntoParts, IParsedUrl } from "./utils";
```

Comment:
Since we're exporting `IParsedUrl` now, please add doc comments to it and its members.  e.g. what is `id` v. `path`?  It's totally unclear by those names.